### PR TITLE
(For Jintae or Gary) Add Session level decisions

### PR DESCRIPTION
**Pull request by mjouahri**
*Purpose:* 
- Adds support for the session level decisions.

*Technical overview:*
- Updates data model to reflect the new decision.

**Testing Plan**
- Expanded and added unit tests and run them locally.

**Deployment**
- Publish to Maven
- Github release

### CODE REVIEW

File: lib/SiftClient.php [hunk 1]
```diff
@@ -469,6 +469,49 @@ public function applyDecisionToOrder($user_id, $order_id, $decision_id, $source,
         return $this->applyDecision($url, $opts);
     }
 
+    /**
+     * Apply a decision to a session. Validates presence of order_id and builds
+     * the url to apply a decision to a session and delegates to applyDecision.
+     *
+     * @param string $user_id the id of order's user id
```
Comment (jintaekim20) [inline lib/SiftClient.php]: `order` should be updated to `session`

File: composer.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "keywords": ["sift", "sift science", "php", "fraud"],
     "homepage": "https://github.com/SiftScience/sift-php",
     "license": "MIT",
-    "version": "2.1.1",
+    "version": "2.2.0",
     "authors": [
```
Comment (jintaekim20) [inline composer.json]: You may add your name in `authors`, too.

**Comment by jintaekim20** [comment]
I was wondering if we also need to update https://github.com/SiftScience/sift-php/blob/db1e02a96fb755a0140ec009b697e35b28b8d7d3/lib/Sift.php#L15 to update version. I don't like that we have another place that has a hard-coded version, but it does seem to be used in `SiftRequest.php`.

File: README.md [hunk 3]
```diff
@@ -121,10 +121,44 @@ $response = $sift->getOrderDecisions('example_order');
...
+```
+$sift = new SiftClient(array('api_key' => 'my_api_key', 'account_id' => 'my_account_id'));
+$response = $sift->applyDecisionToOrder('example_user','example_order','example_decision','example_source',array('analyst' => 'analyst@example.com')
+$response->isOk()
+```
+
+### Apply decision to a session
+```
+$sift = new SiftClient(array('api_key' => 'my_api_key', 'account_id' => 'my_account_id'));
+$response = $sift->applyDecisionToSession('example_user','example_session','example_source',array('analyst' => 'analyst@example.com')
```
Comment (garylee1) [inline README.md]: This example code is missing the `decision` parameter

**Comment by garylee1** [comment]
LGTM after docs fix

**Pull request merged**